custom_components/mqtt_vacuum_camera/camera.py

```diff
@@ -339,7 +338,7 @@ async def handle_obstacle_view(self, event):
     async def _async_find_nearest_obstacle(x, y, obstacles):
         """Find the nearest obstacle to the given coordinates."""
         nearest_obstacle = None
-        min_distance = 500  # Start with a very large distance
+        min_distance = 50  # Start with a very large distance
```

Comment:
@coderabbitai thanks, the comment is now changed to "Start with 50 pixels distance"  float('inf') is not okay because enlarge too much the search range.